CHANGES.rst

```diff
@@ -28,6 +28,9 @@ New Features:
 
 Bug Fixes:
 
+- Use absolute imports for Python3 compatibility
+  [pbauer]
```

Comment:
Leave Philip out of this. *You* did the work. :-)

Reply:
It was a copy/paste gone mad ;)